### ISSUE TITLE
Failed to dkms on linux 5.10

### ISSUE THREAD

**Issue by Hayao0819** [jghodd/uvesafb-dkms#2]
I use it on Arch Linux and when I upgraded Linux kernel to 5.10, dkms command executed by pacman output the error

```

Kernel preparation unnecessary for this kernel.  Skipping...

Building module:
cleaning build area...(bad exit status: 2)
'make' BASEDIR=/usr/lib/modules/5.10.3-zen1-1-zen modules...(bad exit status: 2)
Error! Bad return status for module build on kernel: 5.10.3-zen1-1-zen (x86_64)
Consult /var/lib/dkms/uvesafb/1.0.3/build/make.log for more information.
```

Following log is `/var/lib/dkms/uvesafb/1.0.3/build/make.log`

```
DKMS make.log for uvesafb-1.0.3 for kernel 5.10.3-zen1-1-zen (x86_64)
Thu Dec 31 09:44:36 JST 2020
make -C src/ modules
make[1]: Entering directory '/var/lib/dkms/uvesafb/1.0.3/build/src'
make -C /usr/lib/modules/5.10.3-zen1-1-zen/build M=/var/lib/dkms/uvesafb/1.0.3/build/src modules
make[2]: Entering directory '/usr/lib/modules/5.10.3-zen1-1-zen/build'
  CC [M]  /var/lib/dkms/uvesafb/1.0.3/build/src/uvesafb.o
  MODPOST /var/lib/dkms/uvesafb/1.0.3/build/src/Module.symvers
ERROR: modpost: "vesa_modes" [/var/lib/dkms/uvesafb/1.0.3/build/src/uvesafb.ko] undefined!
make[3]: *** [scripts/Makefile.modpost:111: /var/lib/dkms/uvesafb/1.0.3/build/src/Module.symvers] Error 1
make[3]: *** Deleting file '/var/lib/dkms/uvesafb/1.0.3/build/src/Module.symvers'
make[2]: *** [Makefile:1706: modules] Error 2
make[2]: Leaving directory '/usr/lib/modules/5.10.3-zen1-1-zen/build'
make[1]: *** [Makefile:18: modules] Error 2
make[1]: Leaving directory '/var/lib/dkms/uvesafb/1.0.3/build/src'
make: *** [Makefile:4: modules] Error 2
```

**Comment by jghodd** [comment]
The issue is that the CONFIG_FB_MODE_HELPERS config flag has been shut down for linux 5.10.x. I've submitted a bug report to have it turned back on (https://bugs.archlinux.org/task/69181) but I can't provide any ETA for that fix, if it ever happens. Meanwhile, I modified the source base to instantiate its own copy of vesa_modes, which does fix the issue. The new 1.0.4 version does build and install against linux 5.10.x.
